micro_dl/inference/readme.md

```diff
@@ -14,66 +14,38 @@ If there's not enough memory available on the GPU, and AssertionError will be ra
 be allocated for you.
 
-The following settings can be adjusted in inference using a config file 
-(see example in config_inference.yml and config_inference_2d.yml). The config file consists of 
-three main parts, *images*, *metrics* and *inference_3d*, where the latter only needs to be specified
-if running prediction on 3D images.
-The images part is focused around running predictions, whereas metrics parameters are specified
-if you also would like to generate evaluation metrics. If you've already run inference and generated
-your prediction images you can also evaluate metrics independently using the metrics_script.
+# Config
 
-* model_dir (str): Path to model directory
-* model_fname (str/None): File name of weights in model dir (.hdf5). 
-If left out, latest weights file will be selected.
-* image_dir (str): Directory containing full size input images (not tiles)
-* data_split (str): Which data (train/test/val) to run inference on.
- (default = test)
-* images:
-    * image_format (str): 'zyx' or 'xyz' for depth dimension first or last
-    * flat_field_dir (str/None): Directory containing flatfield images
-    * im_ext (str): For writing generated prediction images, e.g.
-    '.png' or '.npy' or '.tiff'. For 3D images the only option is '.npy'.
-    * crop_shape (list/None): Center crop the image to a specified shape before inference.
-    If None, leave images as is.
-* metrics:
-    * metrics_list (list): List of metrics to estimate. Currently available metrics:
-        * 'ssim' - structural similarity index
-        * 'corr' - correlation
-        * 'r2' - coefficient of determination
-        * 'mse' - mean squared error
-        * 'mae' - mean absolute error
-    * metrics_orientations (list): Assuming images are of shape xyz you can evaluate metrics
-    along any number of given planes 'xy', 'xz', 'yz' as well as generating global 3D metrics
-    using 'xyz'.
-* masks
-    * mask_dir (str): Mask directory containing a frames_meta.csv with
-    mask channels (which will be target channels in the inference config)
-    z, t, p indices matching the ones in image_dir. Mask dirs are often either
-    generated or have had a frames_meta added to them during preprocessing.
-    * mask_type (str): 'target' for segmentation, 'metrics' for weighted metrics
-    * mask_channel (int): mask channel as in training
-* inference_3d: params if doing 3D inference:
-    * num_slices (int): in case of 3D, the full volume will not fit in GPU
-    memory, specify the number of slices to use and this will depend on
-    the network depth, for ex 8 for a network of depth 4.
-    * inf_shape (list): Inference on a center sub volume.
-    * tile_shape (list): Shape of tile for tiling along xyz.
-    * num_overlap (int/list): int for tile_z, list for tile_xyz
-    * overlap_operation (str): e.g. 'mean'
-    
-## Metrics
+> **zarr_dir**: <span style="color:cyan"> absolute path </span> (absolute path to HCS-compatible zarr store containing data)
```

Comment:
@Christianfoley Are these html styles intended to be rendered on GitHub?

Reply:
Github does not render them, however they do render in VSCode. I'm not attached to them, but they don't affect the github rendering either way.

Reply:
I would discourage frequent html tags in markdown. In this case other native markup styles should be sufficient to distinguish these segments